### PR TITLE
Enhancement/donetodomodel jb

### PR DESCRIPTION
**Pull request by jbjoeburns**
model for doneTodos

### CODE REVIEW

File: src/doneTodos/model.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const {DataTypes} = require("sequelize")
+const connection = require("../db/connection")
+
+const DoneTodo = connection.define("DoneTodo", {
+    completedTasks: {
```
Comment (ApacheTech) [inline src/doneTodos/model.js]: The design specs say that this field should be called "todo", rather than "completedTasks" (addDoneTodo.txt).

**Review by ApacheTech — CHANGES_REQUESTED** [review]
Requires a change of name for a field within the model.

File: src/doneTodos/model.js (hunk 1 above)
Comment (jbjoeburns) [reply src/doneTodos/model.js]: I missed that! Thanks for pointing it out! Just changed it now

**Review by ApacheTech — COMMENTED** [review]
Should be good now.

**Review by ApacheTech — APPROVED** [review]
Should be good now.

**Pull request merged**